### PR TITLE
add e2e test for bidirection netem

### PR DESCRIPTION
**Pull request by YangKeao**
An e2e test for #1486

### CODE REVIEW

File: test/e2e/chaos/networkchaos/network_delay.go [hunk 1]
```diff
@@ -257,7 +261,45 @@ func TestcaseNetworkDelay(
...
+	)
+	By("Injecting both direction chaos for 0")
+	err = cli.Create(ctx, bothDirectionNetem.DeepCopy())
+	framework.ExpectNoError(err, "create network chaos error")
+	wait.Poll(time.Second, 15*time.Second, func() (done bool, err error) {
+		result = probeNetworkCondition(c, networkPeers, ports, true)
+		if len(result[networkConditionBlocked]) != 0 || len(result[networkConditionSlow]) != 2 {
+			return false, nil
+		}
+		return true, nil
```
Comment (STRRL) [inline test/e2e/chaos/networkchaos/network_delay.go:292]: How about inverting this `if` statement.

```go
		if len(result[networkConditionBlocked]) == 0 && len(result[networkConditionSlow]) == 2 {
			return true, nil
		}
		return false, nil
```

**Review by STRRL — COMMENTED** [review]
rest LGTM

**Comment by WangXiangUSTC** [comment]
Is this pr should be merged after https://github.com/chaos-mesh/chaos-mesh/pull/1490?

**Comment by STRRL** [comment]
> Is this pr should be merged after #1490?

I think it does not need after #1490.  It seems that this test case does not relate to #1490.

File: test/e2e/chaos/networkchaos/network_delay.go (hunk 1 above)
Comment (YangKeao) [reply test/e2e/chaos/networkchaos/network_delay.go:292]: I prefer to keep the same pattern with other test cases (as it's hard to notice the difference for the next time modifying the test case)

**Review by fewdan — APPROVED** [review]
LGTM

File: test/e2e/chaos/networkchaos/network_delay.go (hunk 1 above)
Comment (STRRL) [reply test/e2e/chaos/networkchaos/network_delay.go:292]: Oh, I just realize it. 👍

**Review by STRRL — APPROVED** [review]
LGTM

**Pull request merged**